devdox/app/services/git_tokens_service.py

```
@@ -130,122 +128,73 @@ def mask_token(token: str) -> str:
+            created_label = await self.label_store.create_new(
+                GitLabel(
+                    label=json_payload.label,
+                    user_id=user_claims.sub,
+                    git_hosting=json_payload.git_hosting,
+                    token_value=encrypted_token,
+                    masked_token=mask_token(json_payload.token_value),
+                    username=retrieved_git_user.get("username", ""),
+                )
+            )
+        except IntegrityError:
+            raise BadRequest(reason=GENERIC_ALREADY_EXIST)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)